src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -134,6 +137,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	}
 
 	ngOnInit() {
+		this.previewFeaturesEnabled = this._configurationService.getValue('workbench.enablePreviewFeatures');
```

Comment:
This is probably fine in this case - but generally it's a good idea to hook into config changes too since right now if someone modified the value they wouldn't see the toolbar until they reopened the notebooks.

Reply:
I noticed this behavior while testing it yesterday. Didn't know if that was standard experience or not. Would you like me to modify it further? If so, where is my starting point for watching config changes?

Reply:
https://github.com/microsoft/azuredatastudio/blob/master/src/sql/workbench/contrib/notebook/browser/notebookThemingContribution.ts#L24

Just hook into that event and then check for the particular setting being changed and update the UI appropriately.

Reply:
Done!